makefiles/vars.inc.mk

```diff
@@ -47,6 +47,7 @@ export CC                    # The C compiler to use.
 export CXX                   # The CXX compiler to use.
 export CCAS                  # The C compiler to use for assembler files, typically the same as CC.
 export CFLAGS                # The compiler flags. Must only ever be used with `+=`.
+export CFLAGS_CPU            # CPU architecture specific compilater options
```

Comment:
well as you asked for it, maybe "CPU architecture specific compiler flags" better matches with the doc of `CFLAGS` above

Reply:
Done.